### DISCUSSION TITLE
Invalid Addresses

### DISCUSSION THREAD

**Discussion by minershaven** [coinfoundry/miningcore#1461] [Ideas]
@oliverw, @seperatis,  I was curious if there was something in place that would prevent miners from using invalid addresses to mine or maybe send those funds to another pool address that's setup to replace any invalid addresses used.  We have a miner that's been using a btc address to mine obtc. He understands that those coins are gone and can't be recovered but something to prevent that would be nice.  If that's something you haven't planned on or been working on, (I know your busy), I wouldn't mind trying to put that together and include it in my next PR.

**Comment by oliverw** [comment]
All addresses get validated by the chain node/daemon before accepting a miner see:

https://github.com/oliverw/miningcore/blob/master/src/Miningcore/Blockchain/Bitcoin/BitcoinPool.cs#L108

https://github.com/oliverw/miningcore/blob/dacc49f47d159a0b75e1862685da7552314e69e2/src/Miningcore/Blockchain/Bitcoin/BitcoinJobManagerBase.cs#L569

**Reply by minershaven** [reply]
this one must have slipped through then since both coins use the same bc1 prefix and address format.
